journal/updated/01-journal.adoc

```diff
@@ -1,146 +1,152 @@
-though he was a great preacher. I thought it would then be a happy day,
-when God would be the teacher of his people, himself;
-that we need not teach every man his neighbour, or his brother, saying,
-"`Know the Lord;`" but that we should all know him, from the least to the greatest.
-This day we knew not then, for all our preaching and long prayers;
-though the Lord did then beget true hungerings and thirstings in our souls after him.
-We had great love and zeal, and desired the knowledge of the truth, as it is in Jesus.
-Sometimes I have said, this was but like Jacob`'s dream, when he awoke and said,
+and I will remember their sin no more.`"
+It seems this man knew not then the day he spoke of,
+though he was a great preacher. I thought it would be a happy day,
+when God Himself would be the teacher of His people,
+so that we need not teach every man his neighbour, or his brother, saying,
+"`Know the Lord;`" but that we would all know Him, from the least to the greatest.
+
+This day we knew not then, despite all our preaching and long prayers,
+though the Lord did then beget true hungerings and thirstings in our souls after Him.
+We had great love and zeal, and desired the knowledge of the truth as it is in Jesus.
+Sometimes I have said this was but like Jacob`'s dream, when he awoke and said,
 "`Surely the Lord is in this place, and I knew it not.`"
-And indeed we knew not the Lord, as we ought to have done; namely, by his light, grace,
+And indeed we knew not the Lord as we ought to have done, namely, by His light, grace,
 and spirit shining in our hearts,
 to give us the light of the knowledge of the Son of God,
 which knowledge keeps a man meek and humble.
 Such are not puffed up in a vain mind,
-to seek after those things that are too high for them,
-as too many are climbing up that way, which is not the way to God the Father;
-the way to the Father is the way of holiness and purity, and humility,
-without which no man shall see the Lord nor enjoy his presence to their comfort.
-
-About this time, being in the year 1656, our ministers told us,
-that there was a sort of people come up in the north, called Quakers,
-that were a people of a strange posture and principles: saying,
-that it was the last days and times that Christ spoke of in the 24th of Matthew,
-"`Many shall come in my name, and deceive many, ver. 5.
+to seek after those things that are too high for them;
+too many are climbing up in that way, which is not the way to God the Father.
+The way to the Father is the way of holiness and purity, and humility,
+without which no man shall see the Lord, nor enjoy His presence to their comfort.
+
+About this time, being in the year 1656, our ministers told us
+that there was a sort of people emerging in the north, called Quakers,
+that were a people of strange practice and principles, saying
+that it was the last days and times that Christ spoke of in the 24th of Matthew:
```

Comment:
maybe: saying that **these were** the last days and times **which** Christ spoke of